applications/dds_video/README.md

```diff
@@ -0,0 +1,96 @@
+# DDS Video Application
+
+The DDS Video application demonstrates how video frames can be written to or
+read from a DDS databus in order to provide flexible integration between
+Holoscan applications and other applications (using Holoscan or not) via DDS.
+
+The application can be run as either a publisher or as a subscriber. In either case,
+it will use the [VideoFrame](../../operators/dds/video/VideoFrame.idl) data topic
+registered by the `DDSVideoPublisherOp` or `DDSVideoSubscriberOp` operators in order
+to write or read the video frame data to/from the DDS databus, respectively.
+
+When run as a publisher, the source for the input video frames will come from an
+attached V4L2-compatible camera via the `V4L2VideoCaptureOp` operator.
+
+When run as a subscriber, the application will use Holoviz to render the received
+video frames to the display. In addition to the video stream, the subscriber
+application will also subscribe to the `Square`, `Circle`, and `Triangle` topics
+as used by the [RTI Shapes Demo](https://www.rti.com/free-trial/shapes-demo).
+Any shapes received by this subscriber will also be overlaid on top of the
+Holoviz output.
+
+![DDS Video Application Workflow](docs/workflow_dds_video_app.png)
+
+## Building the Application
```

Comment:
Does this application target only bare metal for now? Or, are there steps to run inside the HoloHub container, including mounting the RTI Connext bundle?
 
Can you please add a note clarifying that the application is currently tested on x86 only?